Gestao-Energia/Front-end
# SideBar ADMIN component

Task: 
Adds a fixed sidebar to the application. The sidebar includes links to the main sections and is designed to integrate with the existing layout.

How to test:

1. Run the application locally.
2. Check if the sidebar is visible and remains fixed on the screen.
3. Click on the links in the sidebar to confirm they navigate to the correct sections.
4. Test the layout on different screen sizes to ensure proper display and responsiveness.

design layout: https://www.figma.com/design/8jD1yTW5lkpWANIlFAUt5v/Telas-SGE?node-id=127-235&node-type=frame&t=GcZTTrj8FFf2R9tY-0
